app/jobs/regular/user_email.rb

```diff
@@ -164,16 +178,17 @@ def message_for_email(user, post, type, notification, args = nil)
           email_args[:notification_type] = email_args[:notification_type].to_s
         end
 
-        if !SiteSetting.disable_mailing_list_mode && user.user_option.mailing_list_mode? &&
-             user.user_option.mailing_list_mode_frequency > 0 && # don't catch notifications for users on daily mailing list mode
-             (!post.try(:topic).try(:private_message?)) &&
-             NOTIFICATIONS_SENT_BY_MAILING_LIST.include?(email_args[:notification_type])
-          # no need to log a reason when the mail was already sent via the mailing list job
-          return nil, nil
+        # don't catch notifications for users on daily mailing list mode
+        if user.user_option.mailing_list_mode && user.user_option.mailing_list_mode_frequency > 0
```

Comment:
`user.user_option.mailing_list_mode` is already checking for `SiteSetting.disable_mailing_list_mode` so I removed that check here.